### PR TITLE
Ren schuon more words

### PR DESCRIPTION
**Pull request by ElizabethSchuon**
I added some more words, made everything have indexes, and changed many of the nouns to no longer be the default color

### CODE REVIEW

File: database/seeders/BoardSeeder.php [hunk 1]
```diff
@@ -610,26 +669,29 @@ public function run()
...
+                ['id' => $people + 17, 'text' => 'Granparent', 'color' => $default_color],
+                ['id' => $people + 18, 'text' => 'Aunt', 'color' => $default_color],
+                ['id' => $people + 19, 'text' => 'Uncle', 'color' => $default_color],
+                ['id' => $people + 20, 'text' => 'Cousin', 'color' => $default_color],
+                ['id' => $people + 21, 'text' => 'Brother', 'color' => $default_color],
+                ['id' => $people + 22, 'text' => 'Sister', 'color' => $default_color],
+                ['id' => $people + 23, 'text' => 'Sibling', 'color' => $default_color],
+                ['id' => $people + 24, 'text' => 'Niece', 'color' => $default_color],
+                ['id' => $people + 25, 'text' => 'Nephew', 'color' => $default_color],
+                ['id' => $people + 26, 'text' => 'Nibling', 'color' => $default_color],
```
Comment (eloisab17) [inline database/seeders/BoardSeeder.php:694]:
```suggestion
 ['id' => $people + 26, 'text' => 'Sibling', 'color' => $default_color],
```

File: database/seeders/BoardSeeder.php [hunk 2]
```diff
@@ -564,33 +621,35 @@ public function run()
...
-                ['id' => $people + 10, 'Wait Staff' => 'Say', 'color' => $default_color]
+                ['id' => $people + 1, 'text' => 'People', 'color' => $default_color],
+                ['id' => $people + 2, 'text' => 'Teacher', 'color' => $default_color],
+                ['id' => $people + 3, 'text' => 'Classmate', 'color' => $default_color],
+                ['id' => $people + 4, 'text' => 'Friend', 'color' => $default_color],
+                ['id' => $people + 5, 'text' => 'Doctor', 'color' => $default_color],
+                ['id' => $people + 6, 'text' => 'Nurse', 'color' => $default_color],
+                ['id' => $people + 7, 'text' => 'Cashier', 'color' => $default_color],
+                ['id' => $people + 8, 'text' => 'Wait Staff', 'color' => $default_color],
+                ['id' => $people + 9, 'text' => 'Piolot', 'color' => $default_color],
```
Comment (eloisab17) [inline database/seeders/BoardSeeder.php:654]:
```suggestion
                ['id' => $people + 9, 'text' => 'Pilot', 'color' => $default_color],
```

**Comment by eloisab17** [comment]
Looks good. Just a couple of suggestions for spelling.

File: database/seeders/BoardSeeder.php (hunk 1 above)
Comment (lvs-vong) [reply database/seeders/BoardSeeder.php:694]: I think Nibling was intended.
According to Merriam Webster, "Nibling is a gender-neutral term used to refer to a child of one's sibling as a replacement for 'niece' or 'nephew'."

Comment (ElizabethSchuon) [reply database/seeders/BoardSeeder.php:694]: elvis is correct

Comment (ElizabethSchuon) [reply database/seeders/BoardSeeder.php:694]: Sibling is next to brother and sister, nibling was intended per the definition elvis gave

**Pull request merged**